src/axios.js

```diff
@@ -6,3 +6,10 @@ export const api = axios.create({
 		'content-type': 'application/json',
 	},
 });
+
+export const fetchContent = (slug) => (axios.create({
+	baseURL: `/docs/${slug}.md`,
+	headers: {
+		'content-type': 'application/json',
+	},
+}));

```

Comment:
You are right about unnecessary headers object. I think that It's a good practice to place fetches in a separate file even if these are used only in one place, as It's easier to mock them when creating unit tests with Jest.

Reply:
Sure. How about using get method instead of creating new instance inside the body of this arrow function? I guess we won't be using anything other than GET on this path.